cypress/support/commands.js

```diff
@@ -22,4 +22,23 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add(
+    'ouiaId',
```

Comment:
I think this command is available in the frontend-component-utils library for cypress
https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/src/CypressUtils/CustomCommands.js

Reply:
yeah nice